client/spec/helpers/pages.js

```diff
@@ -260,24 +260,40 @@ function AbstractPanelPage(blog) {
     };
 }
 
+DraftsPage.prototype = Object.create(AbstractPanelPage.prototype);
 function DraftsPage(blog) {
     var self = this;
     self._class_name = '.panel--draft';
     AbstractPanelPage.call(self);
 }
-DraftsPage.prototype = Object.create(AbstractPanelPage.prototype);
 
+ContributionsPage.prototype = Object.create(AbstractPanelPage.prototype);
 function ContributionsPage(blog) {
     var self = this;
     self._class_name = '.panel--contribution';
+    self.byFilterBox = by.css('.dropdown-content');
+    self.validFilterButton = element(self.byFilterBox).element(by.buttonText('SELECT'));
     AbstractPanelPage.call(self);
+
+    self.openFiterByMember = function() {
+        return self.column.element(by.buttonText('+')).click();
+    };
+
+    self.filterByMember = function(member_name) {
+        return self.openFiterByMember().then(function() {
+            element(self.byFilterBox)
+            .element(by.css('[data-username="' + member_name + '"]'))
+            .click().then(function() {
+                return self.validFilterButton.click();
+            });
+        });
+    };
 }
-ContributionsPage.prototype = Object.create(AbstractPanelPage.prototype);
 
 function TimelinePage(blog) {
     var self = this;
     self.blog = blog;
-    self.column = element(by.css('.column-timeline'));
+    self.column = element(by.css('.column--timeline'));
```

Comment:
I thought it was implied that why is that in the DOM,

from what I see `--` is an alternative in BEM description, so why?